tests/test_script.py

```diff
@@ -0,0 +1,85 @@
+machine.succeed('export TEZOS_LOG="* -> warning"')
```

Comment:
Probably it's a good idea to start this file with a comment stating that it's supposed to be used only from `tests/tezos-binaries.nix`.